impeller/aiks/canvas.cc

```diff
@@ -300,4 +302,19 @@ void Canvas::DrawTextFrame(TextFrame text_frame, Point position, Paint paint) {
   GetCurrentPass().AddEntity(std::move(entity));
 }
 
+void Canvas::DrawVertices(Vertices vertices,
+                          Entity::BlendMode mode,
+                          Paint paint) {
+  std::shared_ptr<VerticesContents> contents =
+      std::make_shared<VerticesContents>(vertices);
```

Comment:
Use `std::move` here to use the move constructor for `Vertices` and avoid copying the heap buffers of all the `std::vectors` inside.

Reply:
Done